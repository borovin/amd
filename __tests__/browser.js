import test from 'ava'
import Nightmare from 'nightmare'

test.beforeEach(t => {
  t.context.browser = new Nightmare()

  return t.context.browser.goto('http://localhost:3000/')
})

test.afterEach(t => {
  return t.context.browser.end()
})

test('Test browser output', async t => {
  const snapshot = await t.context.browser.wait('h1').evaluate(() => document.querySelector('h1').innerText)

  t.snapshot(snapshot)
})
