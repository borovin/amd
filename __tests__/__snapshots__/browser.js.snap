// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test browser output 1`] = `"a_b_c_text_jquery:3.2.1"`;
